app/Http/Controllers/API/GroupController.php

```diff
@@ -413,32 +414,170 @@ public static function getEventsForGroupv2(Request $request, $idgroups) {
+     *          name="id",
+     *          description="Group id",
+     *          required=true,
+     *          in="path",
+     *          @OA\Schema(
+     *              type="integer"
+     *          )
+     *      ),
+     *      @OA\Response(
+     *          response=200,
+     *          description="Successful operation",
+     *          @OA\JsonContent(
+     *              @OA\Property(
+     *                property="data",
+     *                title="data",
+     *                description="An array of volunteers",
+     *                type="array",
+     *                @OA\Items(
+     *                    ref="#/components/schemas/Volunteer"
+     *                )
+     *              )
+     *          )
+     *       ),
+     *      @OA\Response(
+     *          response=404,
+     *          description="Group not found",
+     *      ),
+     *     )
+     */
+
+    public static function getVolunteersForGroupv2($idgroups) {
         $group = Group::findOrFail($idgroups);
+        $volunteers = $group->allConfirmedVolunteers()->get();
+        return VolunteerCollection::make($volunteers);
+    }
+
+    /**
+     * @OA\Delete(
+     *      path="/api/v2/groups/{id}/volunteers/{iduser}",
+     *      operationId="deleteVolunteerForGroupv2",
+     *      tags={"Groups","Volunteers"},
+     *      summary="Delete Group Volunteer",
+     *      description="Removes a volunteer from a group",
+     *      @OA\Parameter(
+     *          name="id",
+     *          description="Group id",
+     *          required=true,
+     *          in="path",
+     *          @OA\Schema(
+     *              type="integer"
+     *          )
+     *      ),
+     *      @OA\Parameter(
+     *          name="iduser",
+     *          description="User id",
+     *          required=true,
+     *          in="path",
+     *          @OA\Schema(
+     *              type="integer"
+     *          )
+     *      ),
+     *      @OA\Response(
+     *          response=200,
+     *          description="Successful operation",
+     *       ),
+     *      @OA\Response(
+     *          response=404,
+     *          description="Group not found",
+     *      ),
+     *     )
+     */
 
-        // Get the user that the API has been authenticated as.
-        $user = auth('api')->user();
+    public function deleteVolunteerForGroupv2(Request $request, $id, $iduser)
+    {
+        $user = $this->getUser();
 
-        if (!$user || !Fixometer::userHasEditGroupPermission($idgroups, $user->id)) {
-            // We require host permissions to view the list of volunteers.  At the moment this call is only used when
-            // adding volunteers, and this check means we don't have to worry about exposing sensitive data.
-            abort(403);
+        list($name, $area, $postcode, $location, $phone, $website, $description, $timezone, $latitude, $longitude, $country, $network_data, $email) = $this->validateGroupParams(
```

Comment:
Why do we need this line out of interest?

Reply:
We don't.  Thanks.